cms/templates/import.html

```diff
@@ -20,18 +36,30 @@ <h1 class="page-header">
 <div class="wrapper-content wrapper">
   <section class="content">
     <article class="content-primary" role="main">
-
       <div class="introduction">
-          <p>${_("Be sure you want to import a course before continuing. Content of the imported course replaces all the content of this course. {em_start}You cannot undo a course import{em_end}. We recommend that you first export the current course, so you have a backup copy of it.").format(em_start='<strong>', em_end="</strong>")}</p>
-          ## Translators: ".tar.gz" is a file extension, and files with that extension are called "gzipped tar files": these terms should not be translated
-          <p>${_("The course that you import must be in a .tar.gz file (that is, a .tar file compressed with GNU Zip). This .tar.gz file must contain a course.xml file. It may also contain other files.")}</p>
-          <p>${_("The import process has five stages. During the first two stages, you must stay on this page. You can leave this page after the Unpacking stage has completed. We recommend, however, that you don't make important changes to your course until the import operation has completed.")}</p>
+        ## Translators: ".tar.gz" is a file extension, and files with that extension are called "gzipped tar files": these terms should not be translated
+        %if library:
+            <p>${_("Be sure you want to import a library before continuing. Content of the imported library replaces all the content of this library. {em_start}You cannot undo a library import{em_end}. We recommend that you first export the current library, so you have a backup copy of it.").format(em_start='<strong>', em_end="</strong>")}</p>
```

Comment:
Rewrite of the 2nd sentence: "The contents of the imported library will replace the contents of the existing library."
Rewrite of the 4th sentence: "Before you proceed, we recommend that you export the current library, so that you have a backup copy of it."